docs/Hosts/Spawn-Hosts.md

```diff
@@ -54,3 +54,21 @@ lifetime up to 30 days past host creation.
 
 If you'd like to get a notification before a host expires, you can [set up a
 notification](../Project-Configuration/Notifications.md#spawn-host-expiration) for it.
+
+## Hosts Page
+
+The spruce hosts page shows all hosts and offers three batch actions applicable to hosts:
+
+1. Update Status
+   You can force a state change to these statuses:
+   -Decomissioned: Terminate a host running tasks after it's done running its current task.
+   -Quarantined: Stop a host from running tasks without terminating it or shutting it down.
```

Comment:
```suggestion
   - Quarantined: Stop a host from running tasks without terminating it.
```
(q: what makes this different from the plain "Stopped"? It is worth noting here or down below if possible)